### ISSUE TITLE
Please add link to (or instructions on how to get) popup.el

### ISSUE THREAD

**Issue by jpellegrini** [hpdeifel/synosaurus#9]
Hello,

The README says I should get popup.el, but doesn't tell me where to get it. There are a couple of packages in MELPA with names that sound similar to popup, but they don't seem to be the one used in synosaurus.

If I understood correctly, the one used is this: https://www.emacswiki.org/emacs/PopUp
which, as I understand, is the same as this: https://github.com/auto-complete/popup-el.

But the github page says it has been archived and is now read-only; there are 77 forks of it, and I wouldn't know which is more interesting to use (lots of those are probably not maintained, some could be possibly broken...)

So - could you please add to the README a link to the right place where users may get `popup.el`, the correct version?

**Comment by hpdeifel** [comment]
The one in melpa would be this one: https://melpa.org/#/popup

At the time when I wrote this README, popup.el was a well-known package (used e.g. by auto-complete). But since it is now abandoned, the right thing would be to find a modern alternative and implement that in synosaurus.

**Comment by jpellegrini** [comment]
I suppose x-popup-menu (which comes with Emacs) won't do?
(I personally think the old popup.el looks nicer)

https://www.gnu.org/software/emacs/manual/html_node/elisp/Pop_002dUp-Menus.html

**Comment by hpdeifel** [comment]
I think toolkit menus always feel out of place in emacs. The menu used by `company-mode` would be a good fit, but iirc it's not usable in a standalone way.

**Comment by jpellegrini** [comment]
> The menu used by company-mode would be a good fit, but iirc it's not usable in a standalone way.
I agree. But even if it's not, would you think that making synosaurus work with company mode is a good idea? Or too much bloat?

**Comment by jpellegrini** [comment]
Now that I think of it, synosaurus need not depend on company-mode... But it could use it when available (so there's no bloat) -- and it seems pretty easy to add a company backend:

https://github.com/company-mode/company-mode/wiki/Writing-backends

**Comment by hpdeifel** [comment]
I think company mode has a different use case. Synosaurus doesn't really offer completions, but more replacements. Also it should not be triggered while typing (I guess).

Btw, popup.el is also optional. If you configure another `synosaurus-choose-method`, it's not required to be installed. Hint: If you use ivy or helm, `default` may use those.
